utility_code/prune_empty_regions.py

```diff
@@ -32,17 +32,16 @@ def process_region(arg):
             num_entities += chunk.nbt.count_multipath('Entities[]')
         elif isinstance(region, Region):
             # Check if there are entities (fast check)
-            num_entities += chunk.nbt.count_multipath('Entities[]')
+            num_entities += chunk.nbt.count_multipath('Level.Entities[]')
 
             # Check if there are tile entities (fast check)
-            num_block_entities += chunk.nbt.count_multipath('Level.TileEntities[]')
-
-            # Check if there are block entities (fast check)
-            num_block_entities += chunk.nbt.count_multipath('Level.BlockEntities[]')
+            num_block_entities += chunk.nbt.count_multipath('Level.TileEntities[]') # 1.17 and before
+            num_block_entities += chunk.nbt.count_multipath('Level.BlockEntities[]') # Not sure this was ever a thing?
```

Comment:
I don't see it in the change logs for the chunk format, safe to remove this line.

Reply:
Removed